From: Ansgar Burchardt <ansgar@debian.org>
Subject: skip directories starting with a dot when looking for dune.module
--- a/lib/dunemodules.lib
+++ b/lib/dunemodules.lib
@@ -274,7 +274,7 @@ find_modules_in_path() {
         while read m; do
           test -n "$m" && parse_control "$m"
         done <<EOFM
-        $(find -H "$dir" -name $CONTROL | $GREP -v 'dune-[-_a-zA-Z]/dune-[-a-zA-Z_]*-[0-9]\{1,\}.[0-9]\{1,\}/')
+        $(find -H "$dir" -name '.?*' -prune -o -name $CONTROL -print | $GREP -v 'dune-[-_a-zA-Z]/dune-[-a-zA-Z_]*-[0-9]\{1,\}.[0-9]\{1,\}/')
 EOFM
 	  else
         parse_control "$dir"
